### PR TITLE
Master

### PR DESCRIPTION
**Pull request by alexzdn**
Final version of choosing page. Checked both versions of the game. Everything works. Cheers!

### CODE REVIEW

File: src/main/java/controllers/ApplicationController.java [hunk 1]
```diff
@@ -35,6 +35,10 @@ public Result index() {
         return Results.html();
     }
 
+    public Result Choice() {
+        return Results.html().template("views/AcesUp/Choice.html");
+    }
+
```
Comment (giacoppd) [inline src/main/java/controllers/ApplicationController.java:41]: The whitespace burns my soul

File: src/main/java/views/AcesUp/Choice.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<html>
+
+<link href="/assets/css/AcesUp.css" rel="stylesheet">
+<script type="text/javascript" src="/assets/webjars/jquery/2.1.3/jquery.js"></script>
+
+<body>
+<center><h1>Choose the Version</h1></center>
+
+<div class="centered">
```
Comment (giacoppd) [inline src/main/java/views/AcesUp/Choice.html:9]: I thought you were just going to use the center html tag, but way to take the high road

**Comment by giacoppd** [comment]
Good work. That concludes assignment 3

**Comment by erinsullens** [comment]
It all looks good to me

**Comment by giacoppd** [comment]
Assignment 3

**Pull request merged**